plugins/test/unit/plugins/distributors/test_configuration.py

```diff
@@ -3,9 +3,8 @@
 import tempfile
 import unittest
 
-from mock import Mock
```

Comment:
From the PEP8 standpoint - `mock` should be imported after python standard lib(s) and pulp.

Also, not sure what the value is to importing the `mock` package instead of what is being used from it.  Seems that having to prefix everything with `mock.` 
```
@mock.patch()
mock.Mock
```
is just more typing and adds clutter.  Maybe just a personal preference but I would have preferred:
```
- from mock import Mock
+from mock import Mock, patch
```